commit 01ea08b83f9ae446308621c8a403e2dbffb192a9
Author: Miroslav Rezanina <mrezanin@redhat.com>
Date:   Wed Jul 10 15:19:12 2013 +0200

    qga/service-win32.c: diagnostic output should go to stderr
    
    RH-Author: Laszlo Ersek <lersek@redhat.com>
    Message-id: <1372954727-30269-2-git-send-email-lersek@redhat.com>
    O-Subject: [EMBGD RHEL-6.5 qemu-kvm PATCH 1/3] qga/service-win32.c: diagnostic output should go to stderr
    Bugzilla: 976478
    RH-Acked-by: Eric Blake <eblake@redhat.com>
    RH-Acked-by: Petr Matousek <pmatouse@redhat.com>
    RH-Acked-by: Michael S. Tsirkin <mst@redhat.com>
    
    Acked-by: Michael Roth <mdroth@linux.vnet.ibm.com>
    Reviewed-by: Eric Blake <eblake@redhat.com>
    Signed-off-by: Laszlo Ersek <lersek@redhat.com>

diff --git a/qga/service-win32.c b/qga/service-win32.c
index 02926ab..6de8be4 100644
--- a/qga/service-win32.c
+++ b/qga/service-win32.c
@@ -29,7 +29,7 @@ static int printf_win_error(const char *text)
         MAKELANGID(LANG_NEUTRAL, SUBLANG_DEFAULT),
         (char *)&message, 0,
         NULL);
-    n = printf("%s. (Error: %d) %s", text, (int)err, message);
+    n = fprintf(stderr, "%s. (Error: %d) %s", text, (int)err, message);
     LocalFree(message);
 
     return n;
@@ -78,7 +78,7 @@ int ga_install_service(const char *path, const char *logfile,
         SERVICE_DESCRIPTION desc = { (char *)QGA_SERVICE_DESCRIPTION };
         ChangeServiceConfig2(service, SERVICE_CONFIG_DESCRIPTION, &desc);
 
-        printf("Service was installed successfully.\n");
+        fprintf(stderr, "Service was installed successfully.\n");
     } else {
         printf_win_error("Failed to install service");
     }
@@ -111,7 +111,7 @@ int ga_uninstall_service(void)
     if (DeleteService(service) == FALSE) {
         printf_win_error("Failed to delete service");
     } else {
-        printf("Service was deleted successfully.\n");
+        fprintf(stderr, "Service was deleted successfully.\n");
     }
 
     CloseServiceHandle(service);
